### PR TITLE
move mobile cassettes to root spec folder

### PR DESCRIPTION
**Pull request by aherzberg**
## Summary
We currently have our vcr cassettes in the mobile module of vets api. We are the only team that does this, and it requires boilerplate setting/unsetting of the cassette directory any time we have a test that uses cassettes. We already have permission to skip needing other team approval for writing to vets-api/spec/support/vcr_cassettes/mobile so this ticket is to move all of our cassettes to that directly and remove the boilerplate code

## Related issue(s)
https://github.com/department-of-veterans-affairs/va-mobile-app/issues/5713

## Testing done
specs

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: modules/mobile/spec/support/helpers/iam_session_helper.rb
Comment (aherzberg) [inline modules/mobile/spec/support/helpers/iam_session_helper.rb:1]: lint fix demanded by rubocop

File: modules/mobile/spec/support/helpers/mobile_sm_client_helper.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ module MessagingClientHelper
 
     def authenticated_client
       Mobile::V0::Messaging::Client.new(session: { user_id: 123,
-                                                   expires_at: Time.current + 60 * 60,
+                                                   expires_at: Time.current + (60 * 60),
```
Comment (aherzberg) [inline modules/mobile/spec/support/helpers/mobile_sm_client_helper.rb:11]: lint fix demanded by rubocop

**Comment by aherzberg** [comment]
I'm going to request another exception to the LOC limit as all changes are identical.

**Review by Tonksthebear — APPROVED** [review]
Love it. LGTM. Makes sense to have helpers in a helpers folder

**Pull request merged**